src/index.test.js

```diff
@@ -57,6 +57,58 @@ it(`should handle quoted scripts`, () => {
   })
 })
 
+it(`should quote args with spaces`, () => {
+  crossEnv(['cat', '/path with space/file'])
+  expect(crossSpawnMock.spawn).toHaveBeenCalledWith(
+    'cat',
+    ['"/path with space/file"'],
+    {
+      stdio: 'inherit',
+      shell: true,
+      env: process.env,
+    },
+  )
+})
+
+it(`should not quote args without spaces`, () => {
+  crossEnv(['echo', '/path-without-space'])
+  expect(crossSpawnMock.spawn).toHaveBeenCalledWith(
+    'echo',
+    ['/path-without-space'],
+    {
+      stdio: 'inherit',
+      shell: true,
+      env: process.env,
+    },
+  )
+})
+
+it(`should quote args with single quotes`, () => {
+  crossEnv(['cat', "/path'with'quotes"])
+  expect(crossSpawnMock.spawn).toHaveBeenCalledWith(
+    'cat',
+    ['"/path\'with\'quotes"'],
+    {
+      stdio: 'inherit',
+      shell: true,
+      env: process.env,
+    },
+  )
+})
+
+it(`should quote args with double quotes`, () => {
+  crossEnv(['cat', '/path"with"quotes'])
+  expect(crossSpawnMock.spawn).toHaveBeenCalledWith(
+    'cat',
+    ['"/path\\"with\\"quotes"'],
+    {
+      stdio: 'inherit',
+      shell: true,
+      env: process.env,
+    },
+  )
+})
```

Comment:
I need to add a case where the command it self has spaces. It shouldn't be quoted.